tests/plugins/test_import_image.py

```diff
@@ -162,12 +163,17 @@ def test_import_image(namespace, monkeypatch):
     runner.run()
 
 
-def test_import_image_retry(monkeypatch):
+@pytest.mark.parametrize(('retry_delay', 'results'), [
+    (1, [False, True]),
```

Comment:
How about using a small number like: `0.01` ? Things like this really slow down the overall unit testing phase.